interface/themes/style_prism.css

```diff
@@ -11,9 +11,12 @@
 /* Imports area */
 
 /* How to use: font-family: 'Roboto', sans-serif; */
-@import url('https://fonts.googleapis.com/css?family=Roboto');
+@font-face {
+    font-family: "Roboto";
+    src: url("theme_assets/Roboto-Regular.ttf");
```

Comment:
Will that install on a linix computer?

Reply:
@teryhill i dont know but i cant see why not

Reply:
Unicode ttf fonts should be just fine.

Reply:
You can use things that require access to the internet for this.   Local assets only.

Reply:
I do not agree with this.  we can't have imports that reference external sources, live.  This will break systems that are not installed with external internet access for security reasons.   @MigDinny @aethelwulffe @teryhill

Reply:
@tmccormi You are right and I think that was fixed already... not sure